include/revng-c/mlir/Dialect/Clift/IR/CliftOps.td

```diff
@@ -227,10 +227,7 @@ def Clift_SwitchOp : Clift_Op<"switch",
                         Clift_StatementRegion:$default,
                         VariadicRegion<Clift_StatementRegion>:$cases);
```

Comment:
From reading the LLVM source, I can't even find an implementation of this, but further it seems to only look at *operands*, while the array of case values is an *attribute*. That's my understanding of it anyway.

Reply:
It's odd.

On the one hand, CIR's `SwitchOp` only has a single range of variadic operands, and also has an optional array attribute to represent cases. So that gave me the impression that using the `SameVariadicOperandSize` like they did could also be useful for us.

On the other hand, looking at the doc and at other uses, what you say is correct. Which means we can't use that. Thank you anyways.